### ISSUE TITLE
Allow users to use upper case opcodes like `ADD`

### ISSUE THREAD

**Issue by misirov** [huff-language/huff-rs#245]
Related to [Allow users to use PUSH opcode without breaking the compiler](https://github.com/huff-language/huff-rs/issues/244).
Allow users to scream their opcodes out loud. Compiler could translate all uppercase `ADD` instances to lowercase `add`.

```
Error: Unmatched Jump Label
-> ./huff-project-template/src/Test.huff:2446-2449
       |
  > 57 |     ADD                 // [0000000000000000000000000000000000000000000000000000000000000021]
       |
```

**Comment by AbhinavMir** [comment]
Wouldn't text-based non-ambiguity be better in terms of compiler design, in that either keep it `x` or `y`? Probably nitpicking here.

**Comment by jtriley-eth** [comment]
I would argue it would be best to keep it lowercase since uppercase is reserved for user-defined macros, constants, etc. As @AbhinavMir mentioned, this is less ambiguous to do one or the other and toggling caps-lock just for instructions would be slower to write.

**Comment by misirov** [comment]
Fair enough. If this is a standard to follow and it introduces unnecessary complexity then it would make sense to keep it as is.

For context, myself (and others I know) are used to uppercase opcodes and default to them when doing context switching,  so it's more of a dev UX nitpick than anything else. @jtriley-eth
